charts/spire/charts/spire-server/values.yaml

```diff
@@ -175,6 +175,13 @@ ca_subject:
   organization: Example
   common_name: example.org
 
+keyManager:
+  disk:
+    enabled: false
```

Comment:
Looks like the defaults are backwards here, and there is an extra newline at the bottom.

Reply:
sorry about that!

Reply:
No worries. Thanks for fixing it. :)